gapipy/resources/base.py

```diff
@@ -24,6 +24,12 @@ def __init__(self, data, stub=False, client=None):
             self._uri = self._resource_name
         super(Resource, self).__init__(data, client)
 
+    @classmethod
+    def options(cls, client):
+        request = APIRequestor(client, cls)
+        response = request.options()
```

Comment:
might as well do `return requests.options()` here

Reply:
or even shorter:
```
return APIRequestor(client, cls).options()
```

Reply:
fair enough

Reply:
@marz619 Also I'd say it's out of scope for this PR but `create` does this format too as its what I based this function on :wink:

Reply:
can leave it as is... `readability` is good too